resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -590,6 +590,7 @@ codemodernizer.toolwindow.scan_in_progress.stopping=Stopping the job...
 codemodernizer.toolwindow.scan_in_progress.transforming=Amazon Q is transforming your code. Details will appear soon.
 codemodernizer.toolwindow.stop_scan=Stop job
 codemodernizer.toolwindow.table.header.date=Date
+codemodernizer.toolwindow.table.header.job_id=Job Id
```

Comment:
Question: Is there supposed to be a space here or should it be `JobId`? I think there are arguments for both Job Id and JobId, don't want to over complicate something simple.

Reply:
@damntrecky I used "Job Id" because there are spaces in "Module Name" and "Job running time". On that note, VSCode uses different display names: "Id", "Module", and "Duration". While the UIs do not need to look identical across IDEs, this is a straightforward change, and I personally prefer the titles in IntelliJ more as they give more context. Thoughts on changing VSCode to match?